src/options.html

```diff
@@ -34,11 +34,7 @@
 
 			<span class="form__hint">The background colour used while the page is loading.</span>
 
-			<select class="form__input form__input--select" id="theme">
-				<option value="none">None</option>
-				<option value="light">Light</option>
-				<option value="dark">Dark</option>
-			</select>
+			<input class="form__input form__input--textlike" type="color" id="theme" value="#0E0E0E">
```

Comment:
Keeping the 'none' option is important otherwise sites that do not specify a background colour will end up with the custom colour from this extension when displayed in the iframe.